addons/payment_adyen/tests/test_adyen.py

```diff
@@ -523,3 +524,49 @@ def test_no_information_missing_from_partner_address(self):
         test_address = adyen_utils.format_partner_address(test_partner)
         for key in ('city', 'country', 'stateOrProvince', 'street',):
             self.assertTrue(test_address.get(key))
+
+    def test_application_info_in_payment_request(self):
```

Comment:
- Placement of the test method.
- Naming of the method.
- Remove superfluous logic (_get_processing_values, side_effect, url_open, fake response).